### ISSUE TITLE
Mac Builders Error

### ISSUE THREAD

**Issue by LiNk-NY** [lgatto/pRoloc#100]
Hi Laurent, @lgatto 
There are issues with the package on our Mac builders. 

Please see: 
http://bioconductor.org/checkResults/devel/bioc-LATEST/pRoloc/

It seems like this is related to `BioMart`.

```
> getGOFromFeatures(fn, params = dunkley2006params)
Error in getBM(attributes = c(params@filter, attrs, "go_linkage_type"),  : 
  Query ERROR: caught BioMart::Exception::Usage: Attribute go_accession NOT FOUND
Calls: getGOFromFeatures -> getBM
Execution halted
```
Regards,
Marcel

**Comment by lgatto** [comment]
That looks like a biomart issue. I recently had to make changes in my code because attribute names had changed, to address the consistent errors on all platforms. I don't understand why it only fails on Mac here. I am testing locally to see if I can reproduce this. If not, I don't see what I can do.

**Comment by lgatto** [comment]
Ok, seems to be another change in fucking biomart. I'm on it.

**Comment by LiNk-NY** [comment]
Thanks Laurent!

**Comment by lgatto** [comment]
Fixed with commit 0edf3425b97613267bae96a5069564f04864a6b2.
